hw03_gaonaborgesa.py

```diff
@@ -0,0 +1,154 @@
+#################################################################################
+# Author: America Gaona Borges
+# Username: gaonaborgesa
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Continue practicing creating and using functions.
+# More practice on using the turtle library.
+# Learn about how computers represent colors.
+# Learn about source control and Git.
+#
+# Google Doc Link: https://docs.google.com/document/d/1h14KMmSeHQAEvo7u2CwElS8fjyTEb8Z_fG7SychdmWs/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+
+def draw_grass(t):
+    """
+    Makes grass and fills it in.
+
+    :param t: turtle object
+    :return: None
+    """
+    t.penup()              #positions turtle
+    t.setpos(-310,-200)
+    t.pendown()
+
+    for i in range(2):          #makes grass
+        t.forward(610)
+        t.right(90)
+        t.forward(25)
+        t.right(90)
+        t.forward(610)
+        t.left(90)
+        t.forward(25)
+        t.left(90)
+
+
+def draw_sun(t):
+    """
+    Makes sun and sun rays.
+
+    :param t: turtle object
+    :return: None
+    """
+    t.penup()              #positions turtle
+    t.setpos(212, 265)
+    t.pendown()
+    t.right(105)
+
+    for i in range(17):       #sun
+        t.forward(10)
+        t.left(10)
+
+    t.penup()              #sun rays 1/4
+    t.setpos(187, 250)
+    t.pendown()
+    t.left(100)
+    t.forward(25)
+    t.penup()
+
+    t.setpos(195, 210)      #sun rays 2/4
+    t.pendown()
+    t.left(45)
+    t.forward(25)
+    t.penup()
+
+    t.setpos(230, 178)      #sun rays 3/4
+    t.pendown()
+    t.left(35)
+    t.forward(25)
+    t.penup()
+
+    t.setpos(280, 165)      #sun rays 4/4
+    t.pendown()
+    t.left(25)
+    t.forward(25)
+
+
+
```

Comment:
Pay attention to the gray squigglies, they indicate style guidelines. This one refers to there being more three blank lines before a function definition (instead of the recommended 2 blank lines).

Reply:
got it thanks